src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -135,6 +135,12 @@ private void UpdateRequestFlags(Options options)
                     RequestFlags |= AnalyzeRequestFlags.ShowNonPortableApis;
                 }
 
+                //Set OverwriteOutputFile flag if the output file name is explicitly specified
+                if (options.Out != null)
```

Comment:
I'm not sure the best way is to add another `AnalyzeRequestFlag` because these flags are passed to the service and are used to determine what changes we should make to the portability report during analysis.  

I had imaged that an extra property would be added to the [`IApiPortOptions`](https://github.com/Microsoft/dotnet-apiport/blob/master/src/Microsoft.Fx.Portability/IApiPortOptions.cs) interface.

```csharp
public interface IApiPortOptions {
...
    bool OverwriteOutputFile { get; }
}
```